test/edu/csus/ecs/pc2/core/imports/clics/CLICSEventFeedEventTest.java

```diff
@@ -0,0 +1,64 @@
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.HashMap;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.util.AbstractTestCase;
+
+/**
+ * Unit tests.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+public class CLICSEventFeedEventTest extends AbstractTestCase {
+
+    private ObjectMapper mapperField;
+
+    public void testTestNullData() throws Exception {
+
+        ObjectMapper mapper = getMapper();
+        String json = "{\"type\": \"problems\", \"id\": \"542cc6e2-104a-49bc-9fea-04752f9af5ad\", \"op\": \"delete\", \"data\": null }";
+        CLICSEventFeedEvent eventFeedEntry = (CLICSEventFeedEvent) mapper.readValue(json, CLICSEventFeedEvent.class);
+        assertNull(eventFeedEntry.getData());
+
+    }
+
+    public void testTestWithData() throws Exception {
+
+        String eventLines[] = { "{\"type\": \"organizations\", \"id\": \"1add73b6-c66f-46de-a81e-a38773613394\", \"op\": \"delete\", \"data\": {\"id\": \"rsu_ac_in\"}}", //
```

Comment:
I suggest using sample events that 2022-07 compliant.  These samples are old format events: "op", "delete", etc. are part of the old specification.

Reply:
Sure, that is also out of scope.  Being compliant with any spec was not in scope for this issue, it  is
to provide a comparison between the event feed and the pc2 model - as such providing a framework
and for future changes, including processing/handling 2022-07 spec.

Reply:
The samples were what I had 4 weeks ago when I started coding, part of #559 may be to update comments..